cmd/agent/gui/views/templates/generalStatus.tmpl

```diff
@@ -319,16 +319,23 @@
             {{- end }}
             {{- if .inputs }}
             Inputs: {{ range $input := .inputs }}{{$input}} {{ end }}</br>
+            {{- end }}
             BytesRead: {{ .bytes_read }}</br>
             Average Latency (ms): {{ .all_time_avg_latency }}</br>
             24h Average Latency (ms): {{ .recent_avg_latency }}</br>
             Peak Latency (ms): {{ .all_time_peak_latency }}</br>
             24h Peak Latency (ms): {{ .recent_peak_latency }}</br>
-            {{- if .info }}
-            Container Info:</br>
+            {{- if .simpleInfo }}
+            {{- range $key, $value := .simpleInfo }}
+            {{$key}}: {{$value}}</br>
+            {{- end }}
+            {{- end }}
+            {{- if .multiLineInfo }}
+            {{- range $key, $value := .multiLineInfo }}
```

Comment:
Instead of having both `simpleInfo` and `multiLineInfo`, is it possible to have only `multiLineInfo` and check the cardinality of `$value` to display it as a single value or as a multiple values?

Reply:
Ill give that a shot - didn't realize that was possible with the template language :)